src/application/ApplicationInfo.ts

```diff
@@ -12,5 +11,5 @@ export interface ApplicationInfo {
     applicationVersion: string;
     applicationStage: string;
     applicationTags: any;
-
+    applicationResourceName: string;
```

Comment:
As application resource name might be different for each execution (for ex, same web app handle requests concurrently for different endpoints),  `applicationResourceName` should not be here